### PR TITLE
188 export balances error state

### PR DESCRIPTION
**Pull request by mikozet**

### CODE REVIEW

**Review by louilinn — CHANGES_REQUESTED** [review]
Something with the dates is broken

File: src/components/DialogExportStatement.js [hunk 1]
```diff
@@ -201,6 +204,7 @@ const DialogExportStatement = ({ dialogOpen, onCloseHandler }) => {
           </Typography>
           <Box className={classes.dateInputsContainer}>
             <DatePicker
+              format="dd/MM/yyyy"
```
Comment (louilinn) [inline src/components/DialogExportStatement.js:214]: maybe it needs to be DD/MM/YYYY ? possibly. I havent checked docs yet

Comment (mikozet) [reply src/components/DialogExportStatement.js:214]: Hey I checked again and I think it was a bug in preview mode in my libre office. Once I open file it actually shows all records
preview mode:
Screenshot from 2023-05-02 16-32-17

and here is actual file:
Screenshot from 2023-05-02 16-32-31

**Review by juanenrisley — APPROVED** [review]
👍🏿 Everything is allright

**Pull request merged**